### PR TITLE
Completed Tag UI Component

### PR DESCRIPTION
**Pull request by ermalsh**
Completed Tag UI Component for new design Added Story w/ Controls

### CODE REVIEW

File: kibbeh/src/ui/Tag.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+interface TagProps { }
+
+export const Tag: React.FC<TagProps> = ({
+  children,
+}) => {
+  return (
+    <div
+      className={`bg-gray-700 hover:bg-gray-600 text-xs px-2 font-bold text-primary-100 justify-center items-center rounded`}
```
Comment (ajmnz) [inline kibbeh/src/ui/Tag.tsx]: `bg-gray-*` is now `bg-primary-*`. For me it shows a transparent background in the preview

Comment (ajmnz) [reply kibbeh/src/ui/Tag.tsx]: Add also `cursor-pointer`.

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/ui/Tag.tsx (hunk 1 above)
Comment (ermalsh) [reply kibbeh/src/ui/Tag.tsx]: @ajmnz sounds good! Should we update bg-gray-800 in MultipleUsers.tsx:15 and other places bg-gray is referenced? Those show up transparent in preview too.

Comment (ajmnz) [reply kibbeh/src/ui/Tag.tsx]: ~~Sure, go ahead!~~ Wait, where do you see them being transparent? MultipleUsers shouldn't have any background I think

Comment (ermalsh) [reply kibbeh/src/ui/Tag.tsx]: Done!

Comment (ajmnz) [reply kibbeh/src/ui/Tag.tsx]: I think background doesn't do anything there, so we might be better off removing the class.

**Review by ajmnz — APPROVED** [review]

**Pull request merged**